src/services/highcharts-factory/highcharts-factory.svc.coffee

```diff
@@ -17,6 +17,16 @@ angular
           layout: 'vertical'
           align: 'left'
           verticalAlign: 'middle'
+          useHTML: true
+          plotOptions :
+             series:
+               dataLabels:
+                 useHTML: true
```

Comment:
Not sure if you need the `plotOptions` here? I have tested and it seems to work the same without it, and I can't find the API for it here: http://api.highcharts.com/highcharts/legend